man/systemd-factory-reset.xml

```diff
@@ -0,0 +1,174 @@
+  "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<!-- SPDX-License-Identifier: LGPL-2.1-or-later -->
+<refentry id="systemd-factory-reset"
+          xmlns:xi="http://www.w3.org/2001/XInclude">
+
+  <refentryinfo>
+    <title>systemd-factory-reset</title>
+    <productname>systemd</productname>
+  </refentryinfo>
+
+  <refmeta>
+    <refentrytitle>systemd-factory-reset</refentrytitle>
+    <manvolnum>8</manvolnum>
+  </refmeta>
+
+  <refnamediv>
+    <refname>systemd-factory-reset</refname>
+    <refname>systemd-factory-reset-request.service</refname>
+    <refname>systemd-factory-reset-complete.service</refname>
+    <refname>systemd-factory-reset.socket</refname>
+    <refname>systemd-factory-reset@.service</refname>
+    <refpurpose>Request or complete a factory reset operation, or query current factory reset mode</refpurpose>
+  </refnamediv>
+
+  <refsynopsisdiv>
+    <para><filename>/usr/lib/systemd/systemd-factory-reset</filename></para>
+    <para><filename>systemd-factory-reset-request.service</filename></para>
+    <para><filename>systemd-factory-reset-complete.service</filename></para>
+    <para><filename>systemd-factory-reset.socket</filename></para>
+    <para><filename>systemd-factory-reset@.service</filename></para>
+  </refsynopsisdiv>
+
+  <refsect1>
+    <title>Description</title>
+
+    <para><filename>systemd-factory-reset</filename> is a tool that can query the current factory reset
+    state, request factory request operations or complete them.</para>
+
+    <para>Some of the functionality is also available via the
+    <filename>/run/systemd/io.systemd.FactoryReset</filename> Varlink service.</para>
+
+    <para>See <ulink url="https://systemd.io/FACTORY_RESET">Factory Reset</ulink> for an overview of the
+    factory reset logic.</para>
+  </refsect1>
+
+  <refsect1>
+    <title>Commands</title>
+
+    <para>The <filename>/usr/lib/systemd/systemd-factory-reset</filename> executable may also be invoked from the
+    command line, taking one of the following command arguments:</para>
+
+    <variablelist>
+      <varlistentry>
+        <term><option>status</option></term>
+
+        <listitem><para>Report current factory reset state. Reports one of <literal>unsupported</literal> (if
+        the OS does not support a factory reset logic), <literal>unspecified</literal> (if no factory reset
+        was requested, but it wasn't turned off explicitly either), <literal>off</literal> (if the factory
+        reset logic was explicitly turned off via the kernel command line option), <literal>on</literal> (if
+        the factory reset is currently enabled and executed), <literal>complete</literal> (if the factory
+        reset logic ran during the current boot but is complete now), <literal>pending</literal> (if a
+        factory reset has been requested for the next boot).</para>
+
+        <para>Returns with an exit status of 0 if factory state is current not in effect, 10 if it is
```

Comment:
"if factory state is current not" => "if the factory reset state is currently not"